### ISSUE TITLE
Add comments to image_processing.py

### ISSUE THREAD

**Issue by KBx4** [KBx4/Image-recognition-using-TensorFlow#1]
### It would be a great help if anyone adds comments to the above file.

_Workflow:_
- Importing libraries:
  - keras from tensorflow
  - mathplotlib
  - numpy

- Loading fashion_mnist dataset from Keras
- Train-test splitting
- Creating  three-layer neural network using Keras
- Fitting the model with train data
- Evaluating the model using test data
- Printing accuracy
- Making the model predict class of selected image
